cmake/setup_write_config.cmake

```diff
@@ -56,6 +57,10 @@ MACRO(_summary)
   # Only write to summary.log:
   FILE(APPEND ${_log_summary} "${ARGN}")
 ENDMACRO()
+MACRO(_featurelog)
```

Comment:
Can you add a couple of newlines before this to make the beginning of the macro more easy to see with the eye?